facilities/models/facility_models.py

```diff
@@ -471,11 +471,6 @@ class Facility(SequenceMixin, AbstractBase):
     description = models.TextField(
         null=True, blank=True,
         help_text="A brief summary of the Facility")
-    location_desc = models.TextField(
```

Comment:
@jason-wanjohi @owagaantony Watch out for this. Facility location description has move to physical address